code/game/objects/items/implants/implant.dm

```diff
@@ -14,28 +14,28 @@
 /obj/item/implant/proc/trigger(emote, mob/living/carbon/source)
 	return
 
-/obj/item/implant/proc/on_death(emote, mob/living/carbon/source)
-	return
-
 /obj/item/implant/proc/activate()
 	SEND_SIGNAL(src, COMSIG_IMPLANT_ACTIVATED)
 
 /obj/item/implant/ui_action_click()
 	activate("action_button")
 
-/obj/item/implant/proc/can_be_implanted_in(mob/living/target) // for human-only and other special requirements
-	return TRUE
-
-/mob/living/proc/can_be_implanted()
-	return TRUE
-
-/mob/living/silicon/can_be_implanted()
-	return FALSE
+/obj/item/implant/proc/can_be_implanted_in(mob/living/target)
+	// Originally mob types could specify their own "implant permission" code, but it wasn't used
+	// anywhere, and neither was this proc. Folding the two together means you can have an
+	//implant that always works without too much jumping through hoops.
```

Comment:
I don't think we need to document what code *used* to be, we have a lot of this and it loses its relevancy really fast.